scalecodec/type_registry/polymesh.json

```diff
@@ -92,12 +92,35 @@
         ["total_supply", "Balance"],
         ["owner_did", "IdentityId"],
         ["divisible", "bool"],
-        ["asset_type", "AssetType"],
-        ["primary_issuance_agent", "Option<IdentityId>"]
+        ["asset_type", "AssetType"]
```

Comment:
As we remove the variable type then it may affect the old block decoding. I don't remember how we are tackling this issue in here ?

Reply:
The same will happen with the changes to permissions, the external agents are a breaking change